test/index.js

```diff
@@ -27,15 +27,15 @@ describe('Errors', () => {
                 return done();
             }
 
-            expect(result).to.equal({ data: { name: 'Error', stack: 'Some\nstack', message: 'foo' } });
+            expect(result.data.stringifyable).to.equal({ name: 'Error', stack: 'Some\nstack', message: 'foo' });
         });
 
         const err = new Error('foo');
         err.stack = 'Some\nstack';
         stream.end({ data: err });
     });
 
-    it('converts boom wrapped errors to object literals', (done) => {
+    it('add stringifyable property to boom wrapped error objects', (done) => {
```

Comment:
Could you also assert that other boom data properties are retained in the output? So instead do:

```js
err = Boom.wrap(err, { some: 'other', important: 'props' });
```